landoapi/api/revisions.py

```diff
@@ -7,30 +7,24 @@
 """
 from connexion import problem
 from landoapi.phabricator_client import PhabricatorClient
-from landoapi.transplant_client import TransplantClient
 
 
-def get(revision_id, api_key=None):
-    """ API endpoint at /revisions/{id} to get revision data. """
+def get_revision(revision_id, api_key=None):
```

Comment:
I get the idea behind this method, but, in my view it would be better to keep this file to 2 things: 1) the api functions for revisions, 2) helper methods for only those api functions.

If the `transplants.py` needs to get revisions, create a `_get_revision` method within that file specific to its needs.